### PR TITLE
[tracker] Start automatically. Add setting to start/stop it.

### PR DESCRIPTION
**Pull request by Fabien-B**
Tracker start sending data automatically while the drone is in flight.
Also add a setting to start sending data when the drone is not in flight.

### CODE REVIEW

File: sw/airborne/modules/e_identification_fr/e_identification_fr.c [hunk 1]
```diff
@@ -138,8 +139,11 @@ void e_identification_fr_init()
 
 void e_identification_fr_periodic()
 {
-
-  if (gps.fix) {
+  if(autopilot_in_flight()) {
```
Comment (gautierhattenberger) [inline sw/airborne/modules/e_identification_fr/e_identification_fr.c:142]: `autopilot_in_flight` is currently only working on rotorcraft, but fixedwing can use `launch` variable instead (use ifdef `FIXEDWING_FIRMWARE` or `ROTORCRAFT_FIRMWARE`)
see #2641

File: conf/modules/e_identification_fr.xml [hunk 2]
```diff
@@ -6,11 +6,18 @@
     <configure name="E_ID_PORT" value="UARTx" description="UART where the e-id device is connected to (UART1, UART2, etc"/>
     <configure name="E_ID_BAUD" value="B115200" description="UART baud rate"/>
   </doc>
+  <settings>
+    <dl_settings>
+      <dl_settings name="eID">
+        <dl_setting shortname="e_id_running" var="e_identification_started" min="0" step="1" max="1"/>
```
Comment (gautierhattenberger) [inline conf/modules/e_identification_fr.xml:12]: add `values="STOPPED|RUNNING` attribute ?

File: sw/airborne/modules/e_identification_fr/e_identification_fr.c [hunk 3]
```diff
@@ -138,8 +139,11 @@ void e_identification_fr_init()
...
+    e_identification_started = true;
```
Comment (gautierhattenberger) [inline sw/airborne/modules/e_identification_fr/e_identification_fr.c:143]: so no way to stop in flight ? might be better like that I guess

**Review by gautierhattenberger — CHANGES_REQUESTED** [review]

**Comment by OpenUAS** [comment]
A function to start or stop which is callable in a flightplan welcomed

**Review by gautierhattenberger — APPROVED** [review]

**Comment by gautierhattenberger** [comment]
> A function to start or stop which is callable in a flightplan welcomed

This is possible by changing the running state of the module periodic function. As this is a legal requirement, I don't want to make it "too" easy.

**Pull request merged**